src/emulate.c

```diff
@@ -665,30 +666,59 @@ static inline bool op_jal(struct riscv_t *rv, uint32_t insn)
     return false;
 }
 
+/* Get current time in microsecnds and update csr_time register */
+static inline void update_time(struct riscv_t *rv)
+{
+    struct timeval tv;
+    rv_gettimeofday(&tv);
+
+    uint64_t value = (uint64_t) tv.tv_sec * 1e6; /* uint64_t - seconds */
+    value += (uint32_t) tv.tv_usec;              /* microseconds */
```

Comment:
Remove the comments about seconds and microseconds since they are intuitional enough.